spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/StandardizationValidationSuite.scala

```diff
@@ -0,0 +1,63 @@
+package za.co.absa.enceladus.standardization
+
+import org.scalatest.FunSuite
+import za.co.absa.atum.model.{Checkpoint, Measurement}
+
+class StandardizationValidationSuite extends FunSuite {
```

Comment:
Not sure I like this unit test is not clear what it bounds to. 
For discussion: what about extracting the atum related logic to a dedicated `object` (or `class`), make it easier to test. Also `StandardizationJob` is becoming little fatty of a mixture of unrelated code.

Reply:
I like this test suite and I like the idea of putting Standardization prerequisites validation into a separate object. Since
* It might be used in Conformance as well, for instance for jobs that do not require Standardization.
* It is expected that we will add more validations in the near future

When validations are extracted into a separate class or object, this test suite will fit perfectly.

@AdrianOlosutean , sorry that these ideas were not described in the GitHub issue. My bad, I didn't think it through properly.

Reply:
Ok, I will extract it into a separate object

Reply:
`ControlInfoValidation` maybe?

Reply:
For the record, I don't have any issue with the suite itself. 😄 
It's just it's not clear what it unit tests. (e.g. the `Shift`+`Ctrl`+`T` key combination doesn't tap here